src/reducers/select.js

```diff
@@ -0,0 +1,202 @@
+/**
+*
+* @licstart  The following is the entire license notice for the JavaScript code in this file.
+*
+* Merge MARC records
+*
+* Copyright (C) 2015-2019 University Of Helsinki (The National Library Of Finland)
+*
+* This file is part of marc-record-merge-js
+
+* marc-record-merge-js program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Lesser General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* marc-record-merge-js is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*
+* @licend  The above is the entire license notice
+* for the JavaScript code in this file.
+*
+*/
+import createDebugLogger from 'debug';
+import {normalizeSync} from 'normalize-diacritics';
+//export {strictEquality, subsetEquality}; // this does not work either
+export function strictEquality() { ... }
+export function subsetEquality() { ... }
```

Comment:
The `...`s were just placeholder, not literary values. The implementation goes inside the curly braces.